app/components/ui/forms.tsx

```diff
@@ -37,10 +38,23 @@ export function TextArea({ label, ...props }: TextAreaProps) {
 
 type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement>;
 
-export function Button({ ...props }: ButtonProps) {
+export function Button({ children, ...props }: ButtonProps) {
   return (
-    <button {...props} className="border-2 w-24 bg-red-700 text-white rounded-lg" type="submit">
-      Create
+    <button {...props} className="border-2 bg-red-700 text-white rounded-lg" type="submit">
+      Create {children}
     </button>
   );
 }
+
+type LinkButtonProps = {
```

Comment:
we found this method to put `to` into the prop type, but it would be better if there's a react prop type for Link

Reply:
I think Remix exports a `LinkProps` type or similar that we could use here!